site/content/entry/citra-progress-report-2019-q1.md

```diff
@@ -0,0 +1,269 @@
+
+In the past week, [FearlessTobi](https://github.com/FearlessTobi) took an interest in the code for mic, and decided to try and clean it up a bit.
+During his clean-up, he found that rewriting one part of the code actually fixed it!
+[jroweboy](https://github.com/jroweboy) took it from there, finished it and submitted it for review.
+Microphone support is now in Canary builds for user testing, so try it out and let us know if it works for you!
+
+## #8 Background images for the UI
+
+Not too surprising that this is not heavily requested, as it has very niche uses for streamers and YouTuber's who use Citra.
+Either way, if the demand for such a feature increases, we could take a little time to add it, but we have no plans to work on this as it's not a high priority.
+Accurate emulation, followed closely by making that accurate emulation fast, is always the No.1 priority.
+
+****
+**As you might've already noticed, we've also had many other features added to Citra in these past 6 months.
+They weren't covered in a blog article, so let's take a quick glance at all the new additions:**
+
+## [Multiplayer Version 4](https://github.com/citra-emu/citra/pull/4468) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+Thanks to the collaboration of developers Flame Sage and zhaowenlan1779, the new multiplayer update which came a few months ago, brought many improvements to Citra's multiplayer experience.
+A brief overview of these improvements:
+
+* Users, who verify their Citra Web Services account, will now have their username and avatar displayed in public rooms. You can also right-click to view their Citra community profile.
+
+* Room Moderation - Verified Room hosts can now perform basic moderation actions in their own rooms. Hosts can now kick or ban a user, and remove bans if needed.
+Room bans will ban both the user's Citra Web Services account (if authenticated) and IP address.
+
+* Rooms descriptions - Optionally, you can add a description to your room.
+This is accessible both in the UI and on the command line.
+
+* Console ID check - Sometimes multiple Citra users will have the same virtual Console ID - this is not supposed to happen on real consoles, and as such can confuse games a lot.
+A Console ID conflict check will be performed and users won't be permitted to join if they have the same Console ID as someone else in the room.
+
+* When the room is full, instead of a general "Unable to connect", a proper Error message will get displayed.
+
+* You can now see info messages like "joined" and "left" in the chat.
+Additionally, moderation actions like kicking and banning will also be
+shown to all room members.
+
+* User Pinging - You can now ping other users by `@ + Username or Nickname`.
+Pinged users will get a notification on their desktop and the message will be highlighted.
```

Comment:
might as well consider adding a pic of the current room chat dialog here, showing the new features?